Doc/library/ftplib.rst

```diff
@@ -59,25 +59,25 @@ FTP objects
                source_address=None, *, encoding='utf-8')
 
    Return a new instance of the :class:`FTP` class.
-   When *host* is given, the method call :meth:`connect(host) <connect>`
-   is made by the constructor.
-   When *user* is given, additionally the method call
-   :meth:`login(user, passwd, acct) <connect>` is made.
 
    :param str host:
       The hostname to connect to.
+      If given, :code:`connect(host)` is implicitly called by the constructor.
+      See the corresponding parameter to :meth:`connect` for more details.
```

Comment:
Perhaps there is a better way to word this sentence. Ditto for the similarly worded sentences below.